pkg/iac/scanners/terraform/setup_test.go

```diff
@@ -14,6 +15,68 @@ import (
 	"github.com/aquasecurity/trivy/pkg/iac/terraform"
 )
 
+var emptyBucketCheck = `# METADATA
+# schemas:
+# - input: schema.cloud
```

Comment:
```suggestion
# - input: schema["cloud"]
```

Why didn't this test fail if the wrong schema was used?

Reply:
`schema.cloud` is the valid path to the schema. A schema is an object that stores schemas and the fields of the object can be accessed via point or brackets.

https://www.openpolicyagent.org/docs/latest/policy-language/#using-schemas-to-enhance-the-rego-type-checker

Reply:
TIL!

Reply:
TIL!